tests/storage/test_devices.py

```diff
@@ -223,11 +229,16 @@ def test_get_device_updates_by_remote_cross_signing_key_updates(
             self.store.get_device_updates_by_remote("somehost", -1, limit=3)
         )
 
-        # Even though the limit has not been hit, the next update will be duplicated
-        # but there's no space for it here.
+        # Here we expect the device updates for `device_id1` and `device_id2`.
+        # That means we only receive 2 updates this time around.
+        # If we had a higher limit, we would expect to see the pair of
+        # (unstable-prefixed & unprefixed) signing key updates for the device
+        # represented by `fakeMaster` and `fakeSelfSigning`.
```

Comment:
Thanks, this helps a lot. (I thought that `device_1` and `device_2` might be expecting to receive a pair of updates each; don't fully grok the cross-signing machinery.)